product_docs/docs/biganimal/release/getting_started/02_connecting_to_your_cloud/connecting_azure.mdx

```diff
@@ -21,13 +21,13 @@ To connect your cloud:
 1.  Create a BigAnimal CLI credential:
 
    ```shell
-   biganimal create-credential --name <cred> --address portal.biganimal.com --port 443
+   biganimal credential create --name <cred> --address portal.biganimal.com --port 443
```

Comment:
please remove mention of `--address portal.biganimal.com --port 443` flags, as these are for internal use

Reply:
done